samples/bluetooth/hci_uart/Kconfig

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2021 Zephyr Project members and individual contributors
+# SPDX-License-Identifier: Apache-2.0
+
+mainmenu "Bluetooth: HCI UART"
+
+menu "HCI UART"
+
+config HCI_UART_APP_DEFAULT_FEATURES
+	bool "Enable default HCI UART features"
+	imply BT_CTLR_ASSERT_HANDLER
+	imply BT_CTLR_DTM_HCI
+	default y
+
+config BT_CTLR_PRIVACY
+	bool "Enable privacy feature"
+	default n if HCI_UART_APP_DEFAULT_FEATURES
```

Comment:
This is redefining the `BT_CTLR_PRIVACY` Kconfig, and disabling Controller Privacy for all vendors. This is make conformance testing using HCI UART transport fail for Controllers that support privacy.

Controllers that do not support features should have correct values returned in local support features and commands  when a Host requests it using HCI commands.

Why does ESP32 `select BT_CTLR_PRIVACY_SUPPORT`  ? and then disable in this sample?

```suggestion
```